src/main/java/core/basesyntax/utils/convert/csv/CsvConverterImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.utils.convert.csv;
+
+import core.basesyntax.utils.convert.ReportConverter;
+import core.basesyntax.utils.transaction.FruitTransaction;
+import core.basesyntax.utils.transaction.OperationFactory;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class CsvConverterImpl implements ReportConverter {
+    private static final String LINE_SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int PRODUCT_INDEX = 1;
+    private static final int PRODUCT_QUANTITY_INDEX = 2;
+    private final String report;
+
+    public CsvConverterImpl(String report) {
+        this.report = report;
+    }
```

Comment:
Storing the entire report as a String in the `CsvConverterImpl` class may not be the best approach, as it ties the conversion logic to a specific report format. Consider redesigning this class to accept a List of Strings (each representing a line of the CSV) in the `convert` method, which would make the class more flexible and easier to test.